sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceResolution.scala

```diff
@@ -309,6 +322,15 @@ case class DataSourceResolution(
       orCreate = replace.orCreate)
   }
 
+  private def convertDeleteFrom(
+      catalog: TableCatalog,
+      identifier: Identifier,
+      delete: DeleteFromStatement): DeleteFromTable = {
+    val relation = UnresolvedRelation(delete.tableName)
+    val aliased = delete.tableAlias.map { SubqueryAlias(_, relation) }.getOrElse(relation)
```

Comment:
Nit: one-line map expressions should use `(...)` instead of `{...}`, like this:

```scala
delete.tableAlias.map(SubqueryAlias(_, relation)).getOrElse(relation)
```

Reply:
Fixed.